Userland/Libraries/LibEDID/EDID.h

```diff
@@ -439,11 +439,11 @@ class Parser final {
 
     template<typename T>
     requires(IsIntegral<T> && sizeof(T) > 1) T read_le(T const*)
-    const;
+        const;
```

Comment:
Does it really want the `const` here on its own line for these templated member functions? Also in .cpp file.

Reply:
That's what the blind reformat did, manually adding a line break after the requires clause and running the formatter again fixed it.